Header header
float32[] pos_x
float32[] pos_y
float32[] size
int64 blob_count